src/lib/AnnotationControls.ts

```diff
@@ -150,13 +150,15 @@ export default class AnnotationControls {
      */
     public init({ onRegionClick = noop }: Options = {}): void {
         const groupElement = this.controls.addGroup(CLASS_ANNOTATIONS_GROUP);
-        this.controls.add(
+        const regionButton = this.controls.add(
             __('region_comment'),
             this.handleClick(onRegionClick, AnnotationMode.REGION),
             `${CLASS_BOX_CONTROLS_GROUP_BUTTON} ${CLASS_REGION_BUTTON}`,
             ICON_REGION_COMMENT,
             'button',
             groupElement,
         );
+
+        regionButton.setAttribute('data-testid', 'bp-AnnotationsControls-regionBtn');
```

Comment:
Can use `CLASS_REGION_BUTTON` here?

Reply:
I intentionally separated them to avoid changes in the class name affecting the test selector.